src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -182,6 +182,20 @@ public class SetAzureSqlManagedInstance : ManagedInstanceCmdletBase
         [PSArgumentCompleter("None", "1.0", "1.1", "1.2")]
         public string MinimalTlsVersion { get; set; }
 
+        /// <summary>
+        /// Id of the primary user assigned identity
```

Comment:
this should be "User Managed Identity (UMI))" Lets do the same in other places.

Reply:
Fixed here and all other places